src/python/pants/backend/python/goals/export.py

```diff
@@ -109,7 +113,20 @@ async def export_virtualenv(
     return ExportResult(
         f"virtualenv for the resolve '{request.resolve}' (using {min_interpreter})",
         dest,
-        symlinks=[Symlink(venv_abspath, py_version)],
+        digest=pex.digest,
+        post_processing_cmds=[
+            PostProcessingCommand(
+                [
+                    "{digest_root}/requirements.pex/__main__.py",
```

Comment:
Is there a better way to learn the name of the requirements.pex? An os.path.join or modernized version of that would be good too.

Reply:
Fixed, good call.